test/sequential/issuance.issuanceConfig.test.ts

```diff
@@ -0,0 +1,261 @@
+} from "../../utils/eventListeners";
+import { getSudoUser, setupApi, setupUsers } from "../../utils/setup";
+import { Sudo } from "../../utils/sudo";
+import {
+  activateLiquidity,
+  claimRewards,
+  deactivateLiquidity,
+  getLiquidityAssetId,
+  getRewardsInfo,
+} from "../../utils/tx";
+import { User } from "../../utils/User";
+import { Market } from "../../utils/market";
+import { BN_ZERO } from "gasp-sdk";
+import { Issuance } from "../../utils/Issuance";
+import { SequencerStaking } from "../../utils/rollDown/SequencerStaking";
+import { createAnUpdate, Rolldown } from "../../utils/rollDown/Rolldown";
+import {
+  stringToBN,
+  waitBlockNumber,
+  waitForSessionN,
+  waitUntilUserCollatorRewarded,
+} from "../../utils/utils";
+
+jest.spyOn(console, "log").mockImplementation(jest.fn());
+jest.setTimeout(2500000);
+process.env.NODE_ENV = "test";
+
+let sudo: User;
+let token1: BN;
+let liqId: BN;
+let miningSplitBeginning: number;
+let stakingSplitBeginning: number;
+let sequencersSplitBeginning: number;
+const poolValue = new BN(2500000);
+
+async function getRewardsAmount(event: any) {
+  const filterData = JSON.parse(JSON.stringify(event))[0].event.data;
+  //const rewardAmountString = filterData[2].replace("0x", "");
+  return stringToBN(filterData[2]);
+}
+
+beforeAll(async () => {
+  try {
+    getApi();
+  } catch (e) {
+    await initApi();
+  }
+
+  // setup users
+  sudo = getSudoUser();
+
+  await setupApi();
+  setupUsers();
+
+  await Sudo.batchAsSudoFinalized(
+    Assets.FinalizeTge(),
+    Assets.initIssuance(),
+    Assets.mintNative(
+      sudo,
+      (await SequencerStaking.minimalStakeAmount()).muln(100),
+    ),
+  );
+});
+
+beforeEach(async () => {
+  //remember the initial system parameters for recovery after tests
+  const issuanceConfigBefore = await Issuance.getIssuanceConfig();
+  miningSplitBeginning = issuanceConfigBefore.liquidityMiningSplit / 10000000;
+  stakingSplitBeginning = issuanceConfigBefore.stakingSplit / 10000000;
+  sequencersSplitBeginning = issuanceConfigBefore.sequencersSplit / 10000000;
+  //we use integers in the tests to make things simpler
+  await Sudo.batchAsSudoFinalized(await Issuance.setIssuanceConfig(40, 20, 40));
+});
+
+test("Compare amount of mining rewards for 2 difference configuration", async () => {
+  [token1] = await Assets.setupUserWithCurrencies(
+    sudo,
+    [poolValue.muln(2)],
+    sudo,
+  );
+
+  await Sudo.batchAsSudoFinalized(
+    Sudo.sudoAs(
+      sudo,
+      Market.createPool(GASP_ASSET_ID, poolValue, token1, poolValue),
+    ),
+  );
+
+  liqId = await getLiquidityAssetId(GASP_ASSET_ID, token1);
+
+  await Sudo.batchAsSudoFinalized(Assets.promotePool(liqId.toNumber(), 20));
+
+  const [testUser1, testUser2] = setupUsers();
+
+  await Sudo.batchAsSudoFinalized(
+    Assets.mintToken(liqId, testUser1, poolValue),
+    Assets.mintNative(testUser1),
+    Assets.mintToken(liqId, testUser2, poolValue),
+    Assets.mintNative(testUser2),
+  );
```

Comment:
I suggest minting the liquidity to the pool instead of printing and minting the liqId to the users.  This can lead some wrong behaviors,

Reply:
Done!